schemas/context/profile-contact-crm.schema.json

```diff
@@ -0,0 +1,154 @@
+    "profile-contact-crm": {
+      "properties": {
+        "xdm:contactID": {
+          "title": "Contact Record Id",
+          "description": "Contact record id.",
+          "type": "string"
+        },
+        "xdm:sourceType": {
+          "title": "Contact Source Type",
+          "description": "Contact source type.",
+          "type": "string"
+        },		
+        "xdm:ownerID": {
+          "title": "Contact Owner id",
+          "description": "The ID of the owner of the account associated with this contact.",
+          "type": "string"
+        },
+        "xdm:accountID": {
+          "title": "Contact Account id",
+          "description": "ID of the account that’s the parent of this contact.",
+          "type": "string"
+        },
+        "xdm:userOwnerID": {
+          "title": "User Owner Id",
+          "description": "User owner id",
+          "type": "string"
+        },
+        "xdm:deleted": {
+          "title": "Deleted Indicator",
+          "description": "Indicates whether the contact has been deleted or not .",
+          "type": "boolean"
+        },
+        "xdm:emailBouncedDate": {
+          "title": "Email Bounced Date",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the date when the email was bounced.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:emailBouncedReason": {
+          "title": "Email Bounced Reason",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the reason for bounced email.",
+          "type": "string"
+        },
+        "xdm:emailBounced": {
+          "title": "Email Bounced Indicator",
+          "description": "If bounce management is activated and an email is sent to a contact, indicates whether the email bounced (true) or not (false).",
+          "type": "boolean"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:contactDescription": {
+          "title": "Contact Description",
+          "description": "Contact description.",
+          "type": "string"
+        },
+        "xdm:contactStatus": {
+          "title": "Contact Status",
+          "description": "Current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactStatusDetail": {
+          "title": "Contact Status Detail",
+          "description": "Additional Details of the current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactType": {
+          "title": "Contact Type",
+          "description": "Type of the contact.",
+          "type": "string"
+        },
+        "xdm:userID": {
+          "title": "User Id",
+          "description": "User id.",
+          "type": "string"
+        },
+        "xdm:originalReferrer": {
```

Comment:
Don't need this in CRM Mixin.

Reply:
You instructed to move these original* attributes to the crm mixin in one of your feedback emails. Should I remove these now ?